docs/production/authentication-methods.md

```diff
@@ -49,7 +49,11 @@ it as follows:
     * **SP Entity ID**: `https://yourzulipdomain.example.com`.
     * **SSO URL**:
       `https://yourzulipdomain.example.com/complete/saml/`.  This is
-      the "SAML ACS url" in SAML terminology.
+      the "SAML ACS url" in SAML terminology. If you're using
+      a configuration with `SOCIAL_AUTH_SUBDOMAIN` (see documentation on
+      [authentication with multiple organizations](../production/multiple-organizations.html#authentication))
+      this URL should go to the configured subdomain. For example,
+      `https://auth.example.com/complete/saml/` if `SOCIAL_AUTH_SUBDOMAIN="auth"`.
```

Comment:
I think this should be `auth.yourzulipdomain.example.com`, right?

Reply:
Hmm, not sure what's better - in how I wrote it, I assumed the orgs are running on subdomains of ``example.com`` - so ``org1.example.com`` etc. We don't want someone to get confused and put  "auth.org1.example.com".

Reply:
Yeah, it's probably better to just give a quick example; I did that and merged.